uw-frame-components/portal/messages/controllers.js

```diff
@@ -557,7 +557,20 @@ define(['angular'], function(angular) {
           if (actionType == addToHome) {
             var slash = url.lastIndexOf('/') + 1;
             var fName = url.substr(slash);
-            PortalAddToHomeService.addToHome(fName);
+            layoutService.addToHomeWithFname(fName);
+            $rootScope.layout = null;
+            $sessionStorage.layout = null;
+            layoutService.getLayout().then(function(data) {
+              $rootScope.layout = data.layout;
+              if (data.layout && data.layout.length == 0) {
+                $scope.layoutEmpty = true;
+              }
+              actionButton.label='Added To Home';
```

Comment:
Prefer sentence case. I'd go so far as to prefer an actual sentence, because that's what it is, a complete sentence stating what happened. 

```
actionButton.label='Added to home.';
```

Might even be worth getting cute and including in the message what it was that was added to home, especially if a screenreader is going to notice the event and read the new button label.